src/components/BorrowerProfile/LoanDescription.vue

```diff
@@ -1,118 +1,149 @@
 <template>
 	<section>
-		<h2
-			class="tw-text-h2"
-		>
-			{{ firstName }}'s story
-		</h2>
-		<div
-			v-if="partnerName"
-		>
-			<div>
-				{{ moreInfoAboutLoan }}
-			</div>
-			<div
-				v-if="loanAlertText"
+		<div>
+			<h2
+				class="tw-text-h2"
+				v-if="isAnonymizationLevelFull"
 			>
-				<h3
-					class="tw-text-h3"
-				>
-					About {{ partnerName }}:
-				</h3>
+				Story
+			</h2>
+			<h2
+				class="tw-text-h2"
+				v-if="!isAnonymizationLevelFull"
+			>
+				{{ borrowerOrGroupName }}'s story
+			</h2>
+		</div>
+		<div>
+			<section v-if="storyDescription">
 				<p
-					v-for="(paragraph, index) in loanAlertTextParagraphs"
+					v-for="(paragraph, index) in storyDescriptionParagraphs"
 					:key="index"
```

Comment:
There's a possibility the key of `index` will conflict on other loops within this page if they also use `index` for their key.
I'd recommend prefixing the index with a string.
Something like
```:key="`storyDescription-${index}`"```